core/templates/dev/head/player/ConversationSkinDirective.js

```diff
@@ -441,6 +441,13 @@ oppia.directive('conversationSkin', [function() {
             $rootScope.loadingMessage = '';
             $scope.hasFullyLoaded = true;
 
+            // If the exploration is embedded, use as site language the
+            // exploration language.
+            var siteLanguage = oppiaPlayerService.getExplorationLanguageCode();
+            if ($window.GLOBALS.SUPPORTED_SITE_LANGUAGES && $scope.isIframed) {
+              $translate.use(oppiaPlayerService.getExplorationLanguageCode());
```

Comment:
Let's say the exploration language code is not in GLOBALS.SUPPORTED_SITE_LANGUAGES. What happens?